content/en/author/jiaojinlong/_index.md

```diff
@@ -0,0 +1,57 @@
++++
+
+# Display name
+
+name = "Jinlong Jiao"
+
+# Username (this should match the folder name)
+
+authors = ["jiaojinlong"]
+
+# Role/position
+
+role = ""
+
+# Organizations/Affiliations
+
+#   Separate multiple entries with a comma, using the form: `[ {name="Org1", url=""}, {name="Org2", url=""} ]`.
+
+organizations = [ { name = "Tianjin Normal University", url = "http://www.tjnu.edu.cn" } ]
+
+# Short bio (displayed in user profile at end of posts)
+
+bio = "My research interests include affective computing, machine learning and deep learning."
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+
+email = "GoldragonJiao@outlook.com"
+
+# Organizational groups that you belong to (for People widget)
+
+#   Set this to `[]` or comment out if you are not using People widget.
+
+user_groups = ["Current Members"]
+
+# Social/Academic Networking
+
+# For available icons, see: https://sourcethemes.com/academic/docs/widgets/#icons
+
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+
+# Link to a PDF of your resume/CV from the About widget.
+
+# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
+
+# [[social]]
+
+#   icon = "cv"
+
+#   icon_pack = "ai"
+
+#   link = "files/cv.pdf"
+
++++
+
+I am a senior undergraduate software engineering student at Tianjin Normal University, Tianjin, China. I am a key member of the Cognitive and Affective Computing Lab at my TJNU, advised by Prof. Ziping Zhao.More info about me: https://dragonjiao.github.io/
```

Comment:
A `senior` student usually means a forth-year student at collage. I suppose you need another word at here.